integrations-and-supported-tools/mlflow/scripts/mlflow_neptune_plugin.py

```diff
@@ -0,0 +1,93 @@
+# The data set used in this example is from http://archive.ics.uci.edu/ml/datasets/Wine+Quality
+# P. Cortez, A. Cerdeira, F. Almeida, T. Matos and J. Reis.
+# Modeling wine preferences by data mining from physicochemical properties. In Decision Support Systems, Elsevier, 47(4):547-553, 2009.
+
+import logging
+import sys
+import warnings
+from urllib.parse import urlparse
+
+import mlflow
+import mlflow.sklearn
+import numpy as np
+import pandas as pd
+from mlflow.models.signature import infer_signature
+from neptune import ANONYMOUS_API_TOKEN
+from neptune_mlflow_plugin import create_neptune_tracking_uri
+from sklearn.linear_model import ElasticNet
+from sklearn.metrics import mean_absolute_error, mean_squared_error, r2_score
+from sklearn.model_selection import train_test_split
+
+logging.basicConfig(level=logging.WARN)
+logger = logging.getLogger(__name__)
+
+# (Neptune) Create Neptune tracking URI
+neptune_uri = create_neptune_tracking_uri(
+    api_token=ANONYMOUS_API_TOKEN,  # Replace with your own
+    project="common/mlflow-integration",  # Replace with your own
+    tags=["mlflow", "plugin", "script"],  # (optional) use your own
+)
+
+# (Neptune) Use Neptune tracking URI to log MLflow runs
+mlflow.set_tracking_uri(neptune_uri)
+
+
+def eval_metrics(actual, pred):
+    rmse = np.sqrt(mean_squared_error(actual, pred))
+    mae = mean_absolute_error(actual, pred)
+    r2 = r2_score(actual, pred)
+    return rmse, mae, r2
+
+
+if __name__ == "__main__":
+    warnings.filterwarnings("ignore")
+    np.random.seed(40)
+
+    # Read the wine-quality csv file from the URL
+    csv_url = (
+        "https://raw.githubusercontent.com/mlflow/mlflow/master/tests/datasets/winequality-red.csv"
+    )
+    try:
+        data = pd.read_csv(csv_url, sep=";")
+    except Exception as e:
+        logger.exception(
+            "Unable to download training & test CSV, check your internet connection. Error: %s",
+            e,
+        )
```

Comment:
Shouldn't we stop the program here, if it fails to create the `data` object? Something like `sys.exit` or `raise`?